blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/modules/scaaas/ScanAsAServiceModule.java

```diff
@@ -28,11 +39,19 @@ public class ScanAsAServiceModule implements Module {
 
     private final ScanAsAServiceModuleConfig scanAsAServiceModuleConfig;
 
+    private final ArtifactoryPropertyService artifactoryPropertyService;
+
+    private final ArtifactoryPAPIService artifactoryPAPIService;
+
     private final Collection<CancelDecider> cancelDeciders;
 
     public ScanAsAServiceModule(ScanAsAServiceModuleConfig scanAsAServiceModuleConfig,
+            ArtifactoryPropertyService artifactoryPropertyService,
+            ArtifactoryPAPIService artifactoryPAPIService,
             Collection<CancelDecider> cancelDeciders) {
         this.scanAsAServiceModuleConfig = scanAsAServiceModuleConfig;
+        this.artifactoryPropertyService = artifactoryPropertyService;
+        this.artifactoryPAPIService = artifactoryPAPIService;
```

Comment:
Is this the correct naming for this class `ArtifactoryPAPIService` what does the `PAPI` stand for ?.

Reply:
https://www.jfrog.com/confluence/display/JFROG/User+Plugins#UserPlugins-TheArtifactoryPublicAPI(PAPI)